action.yml

```diff
@@ -144,6 +159,13 @@ runs:
         GH_ACTION_VERSION: ${{ env.version }}
       with:
         args: ${{ steps.validate.outputs.command }}
+    - name: Save Rainforest Run ID
+      id: save_run_id
+      if: failure()
+      shell: bash
+      run: |
+        sudo apt-get install libxml2-utils
```

Comment:
I'll update this PR to use the new `--save-run-id` flag once that's shipped https://github.com/rainforestapp/rainforest-cli/pull/442 so that we no longer need to install `libxml2-utils` here.